Update set-up-common-area-phones.md (#7936)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/set-up-common-area-phones.md

For more information, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).

>  [!NOTE]
> > If you are using custom Setup Policies that unpins the calling app, the dial pad will not appear on the Common Area Phone.  
> 
> For more information on Teams Setup Policies, see https://docs.microsoft.com/en-us/microsoftteams/teams-app-setup-policies
> 

Reviewer: ```suggestion
> [!NOTE]
> If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information on Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```

For more information, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).

> [!NOTE]

> > If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information on Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).

Reviewer 2: ```suggestion
> If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information about Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```

> > If you are using custom setup policies that unpins the calling app, the dial pad will not appear on the Common Area Phone. For more information about Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).

Reviewer 2: ```suggestion
> If you are using a custom setup policy that unpins the calling app, the dial pad doesn't appear on the Common Area Phone. For more information about Teams setup policies, see [Manage app setup policies in Microsoft Teams](/microsoftteams/teams-app-setup-policies).
```
